### ISSUE TITLE
Cross-project Parent links not established when migrating with download option 0

### ISSUE THREAD

**Issue by GeethikaKrishnan** [solidify/jira-azuredevops-migrator#1052]
**Description:**

Issue occurs when migrating cross project parent-child links with download option 0.

I have a Project A and a Project B. All the epic links are present in project A and the child items are in project B. I am migrating with download options 0. When I migrate Project B (child links) first and project A (epic links) second, the cross-project parent-child links do not get established. 
At the same time, if I were to migrate A first and then B, the links get established.

**To Reproduce:**

•	Create 2 projects in JIRA and create parent child links between items in two projects
•	Migrate the project with the child links first to ADO.
•	Keeping the itemsjournal file, migrate the second project with the child links to ADO
•	Check if parent child links are getting established in ADO.

**Tool version**
 **v3.0.328**
 Also tried the same migration with version v3.0.420

**Attachments**

-	config.json
[LinkMapping_A.json](https://github.com/user-attachments/files/15783964/LinkMapping_A.json)
[LinkMapping_B.json](https://github.com/user-attachments/files/15783975/LinkMapping_B.json)

-	jira-export-log.txt for A
[jira-export-log-240611-124850.txt](https://github.com/user-attachments/files/15783982/jira-export-log-240611-124850.txt)

-	jira-export-log.txt for B
[jira-export-log-240611-124722.txt](https://github.com/user-attachments/files/15783993/jira-export-log-240611-124722.txt)

- wi-import-log.txt for A
[wi-import-log-240611-124923.txt](https://github.com/user-attachments/files/15784001/wi-import-log-240611-124923.txt)

- wi-import-log.txt for B
[wi-import-log-240611-124816.txt](https://github.com/user-attachments/files/15784008/wi-import-log-240611-124816.txt)

**Comment by Alexander-Hjelm** [comment]
Related to https://github.com/solidify/jira-azuredevops-migrator/issues/745?

**Comment by GeethikaKrishnan** [comment]
If I'm not wrong, #745 seems to be an issue with linking sub-tasks to tasks as there is no defined link mapping.

Here I am trying to migrate with download options 0 so that no other issues (issues not in migration query) get migrated. 
I am trying to migrate two separate projects with cross-linked items.

If these items that are migrated in two phases, and the second phase of the migration contains Epic links to the issues in first phase of the migration, the links do not get established even after both migrations are done.

**Comment by Alexander-Hjelm** [comment]
Ok! Have you seen and followed out guidelines when it comes to migrating multiple projects in sequence with `download-options` = 0? See: https://github.com/solidify/jira-azuredevops-migrator/blob/master/docs/faq.md#scenario-2-multiple-projects

**Comment by GeethikaKrishnan** [comment]
Yes, I've looked into this and as per the instructions, 
I've used separate config files and separate workspace folder for each migration.
I've relocated the itemsJournal.txt file into the workspace folder for the next migration.
I have not used the --force flag.

**Comment by Alexander-Hjelm** [comment]
Ok! We will have to look into it. Sounds like something fishy could be going on here with how links are interpreted. Marking as bug.

I assume that you have mapped the `Child` link type as well as the `Parent` Link type in your config?

**Comment by GeethikaKrishnan** [comment]
Yes. I've attached the config for both the migrations which contains the mappings
